telos-backend/test/integration/calendar_api.test.js

```diff
@@ -0,0 +1,133 @@
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const express = require('express');
+const mongoose = require('mongoose');
+const axios = require('axios');
+const routes = require('../../src/routes');
+const CalendarEvent = require('../../src/models/calendar_event');
+
+let mongod;
+let app;
+let server;
+let event1;
+let event2;
+let event3;
+
+beforeAll(async (done) => {
+  mongod = new MongoMemoryServer();
+
+  const connectionString = await mongod.getUri();
+  await mongoose.connect(connectionString, { useNewUrlParser: true });
```

Comment:
Should add the following options as per warnings to all mongoose connections:
- useFindAndModify: false
- useUnifiedTopology: true

Reply:
Thanks for picking that up - have updated db-helper with this as well